build/scripts/DevEnv/NewDevEnv.ps1

```diff
@@ -0,0 +1,189 @@
+using module .\AppProjectInfo.class.psm1
+using module .\ALGoProjectInfo.class.psm1
+
+<#
+Creates a new dev env.
+#>
+[Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSAvoidUsingPlainTextForPassword', '', Justification = 'local build')]
+[Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSAvoidUsingConvertToSecureStringWithPlainText', '', Justification = 'local build')]
+param(
+    [Parameter(Mandatory = $false)]
+    [string] $containerName = "BC-$(Get-Date -Format 'yyyyMMdd')",
+
+    [Parameter(Mandatory = $false)]
+    [string] $userName = 'admin',
+
+    [Parameter(Mandatory = $false)]
+    [string] $password = 'P@ssword1',
+
+    [Parameter(Mandatory = $true)]
+    [string[]] $projectPaths,
+
+    [Parameter(Mandatory = $false)]
+    [string] $packageCacheFolder = ".artifactsCache"
+)
+
+function InstallBCContainerHelper {
+    if (-not (Get-Module -ListAvailable -Name "BCContainerHelper")) {
+        Write-Host "BCContainerHelper module not found. Installing..."
+        Install-Module -Name "BCContainerHelper" -Scope CurrentUser -AllowPrerelease -Force
+    }
+
+    Import-Module "BCContainerHelper" -DisableNameChecking
+}
+
+function CreateBCContainer {
+    $containerExist = $null -ne $(docker ps -q -f name="$script:containerName")
+
+    if ($containerExist) {
+        Write-Host "Container $script:containerName already exists."
```

Comment:
This is nice! 👏 The only assumption is that you're using the same username/password I suppose?

Reply:
Yeah, didn't want to overthink it. There might be a way to get the credentials from the existing docker container, but I don't see why anyone would use different credentials for different containers.

Reply:
I think that's fine 😄